SSDynamicText/NSAttributedString+SSTextSize.h

```diff
@@ -3,10 +3,10 @@
 //  SSDynamicText
 //
 //  Created by Remigiusz Herba on 28/08/15.
-//
+//  Copyright (c) 2015 Splinesoft. All rights reserved. 
 //
 
-#import <Foundation/Foundation.h>
+@import Foundation;
```

Comment:
Modules are :+1: but the Podfile might need an update. See [this example](https://github.com/splinesoft/SSDataSources/blob/master/SSDataSources.podspec#L14).